src/main/scala/viper/gobra/frontend/info/implementation/typing/ExprTyping.scala

```diff
@@ -329,9 +331,9 @@ trait ExprTyping extends BaseTyping { this: TypeInfoImpl =>
 
     case PLength(op) => isExpr(op).out ++ {
       exprType(op) match {
-        case _: ArrayT | _: SliceT => noMessages
+        case _: ArrayT | _: SliceT | StringT => noMessages
         case _: SequenceT => isPureExpr(op)
-        case typ => error(op, s"expected an array, sequence or slice type, but got $typ")
+        case typ => error(op, s"expected an array, string, sequence or slice type, but got $typ")
```

Comment:
I am just addig this comment to an arbitrary line of code. In Go, can you write `x := &str[i]` for a string `str`? If yes, then this should be forbidden by gobra.

Reply:
I did not add support for indexing strings for now. As such, there is no need to explicitly forbid this.